### PR TITLE
tableschema

### PR DESCRIPTION
**Pull request by jdcarroll**
setup all tables except relations.. coming soon

### CODE REVIEW

File: server/views/index.ejs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<head>
+	
+</head>
+<body>
+	<div id="app">
+		<h1>Gravity</h1>
+	</div>
+
+	<script src="/static/js/main.bundle.js"></script>
+</body>
```
Comment (icarlosmendez) [inline server/views/index.ejs]: Jeff,
This document is spaced at 4 spaces instead of 2. Please refactor and re-push.

File: gulpfile.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (al1271) [inline gulpfile.js]: white space

File: gulpfile.js [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+	sourcemaps = require('gulp-sourcemaps'),
+	gutil = require("gulp-util"),
+	webpack = require("webpack"),
+	WebpackDevServer = require("webpack-dev-server"),
+	webpackConfig = require("./webpack.config.js"),
+	stream = require('webpack-stream'),
+	nodemon = require('gulp-nodemon'),
+	jshint = require('gulp-jshint'),
+	stylish = require('jshint-stylish'),
+	chalk = require('chalk');
```
Comment (al1271) [inline gulpfile.js]: lines up odd most likely editor settings

**Comment by reactivepixel** [comment]
Question to consider that we've already covered. Why are there views on the server?

**Comment by reactivepixel** [comment]
Also, the enhancements to server.js are very focused. I would break them out to a separate file / module before merging in with master. Less headache for other devs to be exposed to.

**Comment by icarlosmendez** [comment]
Brandy Bergh has accepted the challenge of refactoring the server.js file per recommendation from Chris Chapman. I have reassigned this issue to her and moved the item back into In Progress.

File: client/main.jsx [hunk 4]
```diff
@@ -0,0 +1,9 @@
+/*
+*
+* 
+Main React component - this is where we will create the view controller for displaying our dashboard 
+*
+*
+*/
+//You will need to make sure that you have react-dom.
+//With react 14.+ they added this react-dom module to handle dom rendering
```
Comment (al1271) [inline client/main.jsx]: empty file

**Comment by reactivepixel** [comment]
@bmbergh @jdcarroll I think this is a dead pull request and the changes in here are half merged with another branch (what we talked about, why two features should be in two different branches. As a Result I am closing this pull request without merging with master. If there is further development on this branch please submit another request.

**Pull request closed without merging**